src/main/java/model/figure/Square.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public int calculateArea() {
+        return side * side;
+    }
```

Comment:
Consider defining a constant for the magic number used in the calculation of the square's area. For example, you could define a constant to represent the exponent used in the area calculation of a square.